static/js/init/coreInit.js

```
@@ -79,153 +117,201 @@ export function createCoreInitializer({
+        sanitizer,                // Direct arg
+        htmlSanitizer: sanitizer, // Direct arg
+        apiClient: apiRequest,    // Direct arg (fetch function)
+        domAPI,                   // Direct arg
+        domReadinessService,      // Direct arg
+        browserService,           // Direct arg
+        globalUtils,              // Direct arg
         APP_CONFIG,
         logger
       });
       DependencySystem.register('projectListComponent', plc);
     }
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)